src/backend/dx11/src/device.rs

```diff
@@ -633,6 +642,13 @@ impl Device {
         let mut desc: d3d11::D3D11_DEPTH_STENCIL_VIEW_DESC = unsafe { mem::zeroed() };
         desc.Format = info.format;
 
+        if let Some(stencil) = read_only_stencil {
+            desc.Flags = match stencil {
+                true => d3d11::D3D11_DSV_READ_ONLY_DEPTH | d3d11::D3D11_DSV_READ_ONLY_STENCIL,
```

Comment:
there is never a case where we end up with just `d3d11::D3D11_DSV_READ_ONLY_STENCIL`? what if the format is stencil-only, and it's read-only?

Reply:
DX11 doesn't seem to support stencil only formats, at least as far as I can tell.